Tools/LambdaTestTool/src/Amazon.Lambda.TestTool/Runtime/LambdaDefaultsConfigFileParser.cs

```diff
@@ -189,12 +189,47 @@ private static void ProcessYamlServerlessTemplateFunctionBased(LambdaConfigInfo
                 
                 if (handler == null) continue;
                 if (string.IsNullOrEmpty(handler)) continue;
-                
-                
+
+                var events = resourceBody.Children.ContainsKey("events") ?
```

Comment:
Looks like you only have this implemented if using Serverless Framework. We also need equivalent versions in the CloudFormation JSON and YAML parse functions.

Reply:
This solution contains the use case in my team. I think i'll have to look at the bigger picture and consider all the use cases here.